lib/bwu_datagrid.dart

```diff
@@ -333,6 +340,12 @@ class BwuDatagrid extends PolymerElement {
       _finishInitialization();
     }
   }
+  
+  String _generateInstanceID () {
+    /* gonzalopezzi: this ID generation method could be improved 
+     * by using the uuid pub package */
+    return "bwudatagrid_${new DateTime.now().millisecondsSinceEpoch}";
```

Comment:
I would just use a private static field 
```dart
static int _uniqueGridIdCounter = 0;
static String get _nextUniqueGridId => 'id${++_uniqueGridIdCounter}';

final String uniqueGridId = _nextUniqueGridId;
```

This makes the above comment redundant.